src/views/SizePage.vue

```diff
@@ -4,10 +4,26 @@
       class="mx-auto flex h-full flex-col items-center justify-between bg-gray_00-light text-gray_05-light dark:bg-gray_00-dark dark:text-gray_05-dark sm:w-screen md:w-96"
     >
       <ProgressNavBar prevPage="/" pageName="size" />
+      <div v-if="isLoading">로딩중입니다.</div>
+      <SizeBoard :sizes="data?.sizes ?? []" v-else />
+      <NextButton
+        :onClick="() => pushPage('ingredient')"
+        isPrimary
+        :disabled="!userSelect.sizeId"
+        >다음</NextButton
+      >
```

Comment:
이런 작은 부분도 가독성을 위해 린트를 설정해두거나 하면 좋을거같아요
```suggestion
        >
        다음
        </NextButton>
```